scapy/contrib/ikev2.py

```diff
@@ -99,6 +101,19 @@
                                                  "ESN":   1,  }, 0),
                          }
 
+IKEv2AuthenticationTypes = {
+  0 : "Reserved",
+  1 : "RSA Digital Signature",
```

Comment:
Indenting with 2 spaces is quite non-standard, but the file was already like that...

Maybe we should fix the indentation in a separate commit **before** this commit series.